book/analytics/client_side_api/client_side_api.md

```diff
@@ -59,49 +62,53 @@ Some characteristics of users should be stored as user properties[c]. This is pa
 
 ### Android:
-
-// Capture author info & user status  
-Map<String, String> articleParams = new HashMap<String, String>();  
-//param keys and values have to be of String type  
-articleParams.put("Author", "John Q");  
-articleParams.put("User Status", "Registered");  
-// sending event parameter  
-AnalyticsAgentUtil.logEvent("Article Read", articleParams);
+```
+  // Capture author info & user status  
+  Map<String, String> articleParams = new HashMap<String, String>();  
+  //param keys and values have to be of String type  
+  articleParams.put("Author", "John Q");  
+  articleParams.put("User Status", "Registered");  
+  // sending event parameter  
+  AnalyticsAgentUtil.logEvent("Article Read", articleParams);
+```
 
 ### iOS:
-
-NSDictionary \*analyticsDictionary = @{@”Author” : @”"John Q", @User Status": @"Registered"};  
-[APAnalyticsManager trackEvent:@"Article Read" withParameters:analyticsDictionary];
+```
+  NSDictionary \*analyticsDictionary = @{@”Author” : @”"John Q", @User Status": @"Registered"};  
+  [APAnalyticsManager trackEvent:@"Article Read" withParameters:analyticsDictionary];
+```
 
 ## Capture Event Duration:
 
 A developer can also add the dimension of time to any Event that s/he tracks. Morpheus will automatically record the duration of the Event, which enables analysts to run analysis on characteristics like average duration for an Event overall, by session and by user. This is useful for events like "Play VOD Item" or "Feed Session"
 You can capture Event duration (along with the Event and its properties) with a single log following this pattern:
 
 ### Android:
-
-// Capture author info & user status
-Map<String, String> articleParams = new HashMap<String, String>();  
-articleParams.put("Author", "John Q");  
-articleParams.put("User Status", "Registered");  
-//Log the timed event when the user starts reading the article  
-//setting the third param to true creates a timed event  
-AnalyticsAgentUtil.logTimedEvent("Article Read", articleParams);
-
-// ...  
-// End the timed event, when the user navigates away from article  
-AnalyticsAgentUtil.endTimedEvent("Article Read");
+```
+  // Capture author info & user status
+  Map<String, String> articleParams = new HashMap<String, String>();  
+  articleParams.put("Author", "John Q");  
+  articleParams.put("User Status", "Registered");  
+  //Log the timed event when the user starts reading the article  
+  //setting the third param to true creates a timed event  
+  AnalyticsAgentUtil.logTimedEvent("Article Read", articleParams);
+
+  // ...  
+  // End the timed event, when the user navigates away from article  
+  AnalyticsAgentUtil.endTimedEvent("Article Read");
+```
 
 ### iOS:
+```
+  NSDictionary \*analyticsDictionary = @{@”Author” : @”"John Q", @User Status": @"Registered" };  
```

Comment:
is the `\` in the `NSDictionary \*analyticsDictionary` is necessary?

Reply:
👍